Telegram_bot-master/bot.rb

```diff
@@ -0,0 +1,38 @@
+require 'rubygems'
+require 'telegram/bot'
+require 'logger'
+require 'pry'
+
+require_relative 'commands.rb'
+require_relative 'token.rb'
+
+def get_current_weather(city)
+  options = { units: 'metric', APPID: OPEN_WEATHER_TOKEN }
+  req = OpenWeather::Current.city(city, options)
```

Comment:
what if this request fail? you should think about such things working with third party services!